### PR TITLE
Raw generic use

### PR DESCRIPTION
**Pull request by talbenzur**

### CODE REVIEW

File: src/main/java/projectManagement/controller/BoardController.java [hunk 1]
```diff
@@ -80,25 +80,25 @@ public ResponseEntity<BaseResponse<Board>> deleteStatus(@RequestAttribute Status
...
     /**
      * get userEmail to authentication , and board id- and return the board
      * @param id
      * @param userEmail
-     * @return Board
+     * @return Long
      */
     @GetMapping("/get/{id}")
-    public ResponseEntity<BaseResponse<Board>> getBoard(@PathVariable Long id,
+    public ResponseEntity<BaseResponse<Long>> getBoard(@PathVariable Long id,
```
Comment (rachelgitovich) [inline src/main/java/projectManagement/controller/BoardController.java:94]: why did you change it to long? it should be board

File: src/main/java/projectManagement/controller/BoardController.java [hunk 2]
```diff
@@ -119,16 +119,16 @@ public ResponseEntity<BaseResponse<List<String>>> getUsers(@PathVariable Long id
      *  share the board with other user.
      * @param dto
      * @param userEmail
-     * @return UserInBoard
+     * @return String
      */
     @PostMapping("share")
-    public ResponseEntity<BaseResponse<UserInBoard>> share(@RequestAttribute ShareBoardDto dto, @RequestAttribute String userEmail) {
+    public ResponseEntity<BaseResponse<String>> share(@RequestAttribute ShareBoardDto dto, @RequestAttribute String userEmail) {
```
Comment (rachelgitovich) [inline src/main/java/projectManagement/controller/BoardController.java:125]: here also, returned objects should remain as they were

**Review by rachelgitovich — CHANGES_REQUESTED** [review]
The returned objects should remain as they were

File: src/main/java/projectManagement/controller/BoardController.java (hunk 1 above)
Comment (talbenzur) [reply src/main/java/projectManagement/controller/BoardController.java:94]: the objectthat we return is id- Long. 
its ok i didn't change the object that we return

File: src/main/java/projectManagement/controller/BoardController.java (hunk 2 above)
Comment (talbenzur) [reply src/main/java/projectManagement/controller/BoardController.java:125]: also, we return the userEmail which is String

**Review by rachelgitovich — APPROVED** [review]

**Pull request merged**